### PR TITLE
Add underglow compatible keymap for Ergo42

### PR DESCRIPTION
**Pull request by Biacco42**

### CODE REVIEW

File: keyboards/ergo42/rev1/config.h [hunk 1]
```diff
@@ -67,9 +67,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* ws2812 RGB LED */
 #define RGB_DI_PIN D3
 #define RGBLIGHT_TIMER
-#define RGBLED_NUM 12    // Number of LEDs
+#define RGBLED_NUM 14    // Number of LEDs
```
Comment (drashna) [inline keyboards/ergo42/rev1/config.h]: This should be redefined in your keymap.  You can "`#undef`" it first, and then define it.

File: keyboards/ergo42/rev1/config.h [hunk 2]
```diff
@@ -67,9 +67,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
 #define ws2812_PORTREG  PORTD
 #define ws2812_DDRREG   DDRD
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/ergo42/rev1/config.h]: These should be added to your keymap's config.h, not globally

**Comment by Biacco42** [comment]
I've moved RGB configurations from global `config.h` to specific keymap's `config.h`

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**